components/scream/src/share/atm_process/SCDataManager.hpp

```diff
@@ -0,0 +1,105 @@
+#ifndef SCREAM_SC_DATA_MANAGER_HPP
+#define SCREAM_SC_DATA_MANAGER_HPP
+
+#include "share/scream_types.hpp"
+#include "ekat/ekat_assert.hpp"
+
+namespace scream {
+
+// This struct provides a intermediary between the AD and the SurfaceCouplingImport/Export AtmophereProcess
+// classes, allowing SCREAM to access CPL data pointers and info about the fields.
+struct SCDataManager {
+
+  template<typename DevT, typename DataT>
+  using view_1d = typename KokkosTypes<DevT>::template view_1d<DataT>;
+  template<typename DevT, typename DataT>
+  using view_2d = typename KokkosTypes<DevT>::template view_2d<DataT>;
+
+  using name_t = char[32];
+
+  SCDataManager() {}
```

Comment:
Since you do it for the destructor, you might use the "default" keyword here as well.